floodviz/static/css/style.css

```diff
@@ -55,12 +51,40 @@ path {
 	width: .3%;
 	stroke: #7fff00;
 }
+
 .cir {
 	fill: #7FFF00;
 }
+
 .nv-indexLine {
 	display: none;
 }
+
+/* Voronoi Stuff */
+
+.rivers {
```

Comment:
This styling is all overridden by the `path` rule above (line 10). I think it would be better to remove that rule and style different classes of paths individually.